client/asset/eth/eth.go

```diff
@@ -2292,6 +2294,19 @@ func (eth *baseWallet) SyncStatus() (bool, float32, error) {
 	return eth.node.peerCount() > 0, 1.0, nil
 }
 
+// TransactionFeesPaid returns exactly how much gwei was used to send a
+// transaction. The transaction must be already mined for this function to work.
+// Returns asset.CoinNotFoundError for unmined txn.
+func (eth *baseWallet) TransactionFeesPaid(ctx context.Context, coinID dex.Bytes) (uint64, error) {
+	var txHash common.Hash
+	copy(txHash[:], coinID)
+	fees, err := eth.node.transactionFeesPaid(ctx, txHash)
```

Comment:
I think this should wait for a few additional confirmations on the transaction just in case.

Reply:
Sure, how many confirms?

Reply:
Let's use 10. Same as redeems.

Reply:
Set to 10.